nixos/modules/services/networking/unbound.nix

```diff
@@ -12,9 +12,17 @@ let
 
   interfaces = concatMapStrings (x: "  interface: ${x}\n") cfg.interfaces;
 
-  forward = optionalString (length cfg.forwardAddresses != 0)
-    "forward-zone:\n  name: .\n" +
-    concatMapStrings (x: "  forward-addr: ${x}\n") cfg.forwardAddresses;
+  isLocalAddress = x: substring 0 9 x == "127.0.0.1";
```

Comment:
It can be ipv6. As I understand it, the value has to be an ip address, so no localhost. I'll extend `isLocalAddress` to accomodate ipv6 addresses for completeness.